src/e2e/specs/dashboard.spec.ts

```diff
@@ -268,37 +273,78 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard  - Payment`, ()
       },
     );
 
-    // verify subscription dialog elements
-    await dashboardPage.subscribeButton.click();
-    await expect(dashboardPage.subscribeDialogCloseButton).toBeVisible();
-    await expect(dashboardPage.yearlyMonthlyTablist).toBeVisible();
-
-    // verify user purchase choices
-    await dashboardPage.yearlyTab.click();
-    await expect(
-      dashboardPage.subscribeDialogSelectYearlyPlanLink,
-    ).toBeVisible();
-
-    await dashboardPage.monthlyTab.click();
-    await expect(
-      dashboardPage.subscribeDialogSelectMonthlyPlanLink,
-    ).toBeVisible();
-
-    // Check monthly redirection
-    await dashboardPage.subscribeDialogSelectMonthlyPlanLink.click();
-    await purchasePage.subscriptionHeader.waitFor();
-    await expect(purchasePage.planDetails).toContainText("monthly");
-    await expect(purchasePage.subscriptionHeader).toBeVisible();
-
-    // Check yearly redirection
-    await page.goto(`${process.env.E2E_TEST_BASE_URL}`);
-    await dashboardPage.subscribeButton.waitFor();
     await dashboardPage.subscribeButton.click();
-    await dashboardPage.subscribeDialogSelectYearlyPlanLink.click();
-    await purchasePage.subscriptionHeader.waitFor();
-    // Text contains invisible characters which need to be hardcoded in an assertion
-    await expect(purchasePage.planDetails).toContainText(
-      `${process.env.E2E_TEST_ENV === "prod" ? "yearly" : "every ⁨2⁩ months"}`,
+    await dashboardPage.verifyPremiumUpsellModalOptions();
+  });
+});
+
+test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Breaches Scan, Continuous Protection, Data Profile Actions`, () => {
+  test.use({ storageState: { cookies: [], origins: [] } });
+
+  test.beforeEach(async ({ landingPage, page, authPage, welcomePage }) => {
+    test.slow();
```

Comment:
```suggestion
    // this test runs through the welcome scan flow, increasing timeout to address it
    test.slow();
```

Reply:
conversely, we can set the `description` within the `.slow()` annotation so it shows up in reports. Per https://playwright.dev/docs/api/class-test#test-slow

```suggestion
    test.slow(true, "this test runs through the welcome scan flow, increasing timeout to address it");
```